website/docs/silk.net/experimental-feed.md

```diff
@@ -0,0 +1,66 @@
+---
+{
+    "TableOfContents": {
+        "Name": "Experimental Feed",
+        "Url": "experimental-feed.html"
+    }
+}
+---
+
+# Experimental Feed
+
+<?# Warning "The experimental feed is not recommended for use for anything beyond playing around with the new features. We don't officially support these builds as they may be unstable, and should not be used in production." /?>
+
+<?# Note "Changes in the experimental feed happen rapidly. We recommend joining the [Silk.NET Discord server](https://discord.gg/DTHHXRt) so that you can keep up with development." /?>
+
+## Configure your project
+
+Right now, your project should look something like this:
+
+```xml
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net5.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Silk.NET" Version="1.4.0" />
+  </ItemGroup>
+
+</Project>
+```
+
+In order to use the experimental feed, you must change this project file slightly. Add the following line to your project:
+
+```xml
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net5.0</TargetFramework>
+    <!-- Add the experimental feed as a restore source -->
+    <RestoreSources>$(RestoreSources);https://dotnet.github.io/Silk.NET/nuget/experimental/index.json</RestoreSources>
+  </PropertyGroup>
+```
+
+## Install an experimental version
+
+Now .NET has access to the experimental feed, it's time to install an experimental package. Check for the latest version on our [GitLab Package Registry](https://gitlab.com/silkdotnet/Silk.NET/-/packages).
+
+At the time of writing, the latest version is `2.0.0-build97.0`. Now to install this package, you can use your IDE's built-in NuGet client, `dotnet package add`, or just modify the project file again like so:
```

Comment:
These version numbers should probably be updated, this seems to be copied from very old docs (pre 2.0 release), and should not point to a out of support TFM

Reply:
I think it’s best to leave the version numbers as is, or rather, it makes no difference as whatever build number we put here the new experimental feed won’t have. Will fix the TFM for sake of argument.

Reply:
Fixed the TFM, left version number as is, which thinking about it is probably better because it'll stick out like a sore thumb that the reader does not want this, whereas if we use a current build number they're more likely to be confused as to why this isn't on the experimental feed.